content/docs/alerting/alertmanager.md

```diff
@@ -6,196 +6,101 @@ nav_icon: sliders
-  }
-}
-```
-
-#### Generic Webhook
-
-The Alertmanager supports sending notifications as JSON to arbitrary
-URLs. This could be used to perform automated actions when an
-alert fires or integrate with a system that the Alertmanager does not support.
-
-```
-notification_config {
-  name: "alertmanager_webhook"
-  webhook_config {
-    url: "http://example.org/my/hook"
-  }
-}
-```
-
-An example of JSON message it sends is below.
-
-```json
-{
-   "version": "1",
-   "status": "firing",
-   "alert": [
-      {
-         "summary": "summary",
-         "description": "description",
-         "labels": {
-            "alertname": "TestAlert"
-         },
-         "payload": {
-            "activeSince": "2015-06-01T12:55:47.356+01:00",
-            "alertingRule": "ALERT TestAlert IF absent(metric_name) FOR 0y WITH ",
-            "generatorURL": "http://localhost:9090/graph#%5B%7B%22expr%22%3A%22absent%28metric_name%29%22%2C%22tab%22%3A0%7D%5D",
-            "value": "1"
-         }
-      }
-   ]
-}
-```
-
-This format is subject to change.
+The Alertmanager handles alerts sent by client applications such as the 
+Prometheus server. It takes care of deduplicating, grouping, and routing
+them to the correct receiver integration such as email, PagerDuty, or OpsGenie.
+It also takes care of silencing and inhibition of alerts.
+
+The following describes the core concepts the Alertmanager implements. Consult
+the [configuration documentation](../configuration) to learn how to use them
+in more detail.
+
+## Grouping
+
+Grouping categorizes alerts of similar nature into a single notification. This
+is especially useful during larger outages when many systems fail at once and
+hundreds the thousands of alerts may be firing simultaniously.
+
+**Example:** Dozens or hundreds of instances of a service are running in your
+cluster when a network partition occurs. Half our your service instances
+can no longer reach the database.
+Alerting rules in Prometheus were configured to send an alert for each service
+instance if it cannot communicate with the database. As a result hundreds of
+alerts are sent to Alertmanager.
+
+As a user one only wants to get a single page while still being able to see
+exactly which service instances were affected. Thus one can configure
+Alertmanager to group alerts by their cluster and alertname so it sends a
+single compact notification.
+
+Grouping of alerts, timing for the grouped notifications, and the receivers
+of those notificiations are configured by a routing tree in the configuration
+file.
+ 
+## Inhibition
+
+Inhibition is a concept of surpressing notifications for certain alerts if
+certain other alerts are already firing.
+
+**Example:** An alert is firing that informs that an entire cluster is not
+reachable. Alertmanager can be configured to mute all other alerts concerning
+this cluster if that particular alert is firing.
+This prevents hundreds to thousands of alerts firing unrelated to the actual
```

Comment:
The alerts are still firing, but notifications are inhibited. Maybe, "This prevents notifications for hundreds or thousands of firing alerts that are unrelated to the actual issue."?